blueprints/module-unification-app/index.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const uniq = require('ember-cli-lodash-subset').uniq;
+const walkSync = require('walk-sync');
+const stringUtil = require('ember-cli-string-utils');
+
+const FileInfo = require('../../lib/models/file-info');
+
+module.exports = {
+  description: 'Generates an Ember application with a module unification layout.',
+
+  filesToRemove: [],
+
+  ignoredFilePaths: [
```

Comment:
I feel like `ignoredFilePaths` wants to be a part of `fileMap`, each key with a `null` value. Unfortunately  `Blueprint` model exects `fileMap` values to be string only.